### ISSUE TITLE
zig build hangs when no build.zig file exists

### ISSUE THREAD

**Issue by Nomy1** [ziglang/zig#7733]
Go to any a path without a build.zig file and run `zig build`

## Expected
Helpful error message

## Result
terminal hangs. I have to Control+C to kill it

## Environment
MacOS Catalina 10.15.5, iTerm

**Comment by LemonBoy** [comment]
It prints this nice error message here:
```
info: Initialize a 'build.zig' template file with `zig init-lib` or `zig init-exe`,
or see `zig --help` for more options.
error: No 'build.zig' file found, in the current directory or any parent directories.
```

This problem was fixed for the 0.7.1 release IIRC.

**Comment by Nomy1** [comment]
Sorry, you are correct. I discovered I was running 0.7.0
